src/order.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default.html
+---
+
+<h2>Order</h2>
+<p>Please use this to order biscuits!</p>
+<form action="/action_page.php">
+    <fieldset>
+      <legend>Order form:</legend>
+      <label for="name">Name:</label><br>
+      <input type="text" id="name" name="name" value="John"><br>
```

Comment:
Because you've added default `value` attributes to this form, any fields I don't fill out, will be saved with these defaults. I would recommend deleting them from each `input` below.